### PR TITLE
Introduce `enableLongApplyMethod` as an option.

### PR DESCRIPTION
**Pull request by Asamsig**
Hi, as discussed on Gitter, I think this could be a useful option for some people.

### CODE REVIEW

File: scalajs/src/main/scala/org/scalablytyped/converter/internal/scalajs/flavours/GenCompanions.scala [hunk 1]
```diff
@@ -52,7 +55,9 @@ final class GenCompanions(findProps: FindProps) extends TreeTransformation {
 
               case Res.One(_, props) if props.isEmpty => Empty
               case Res.One(_, props) =>
-                val requiredProps = props.filter(_.optionality === Optionality.No)
+                val requiredProps =
```
Comment (oyvindberg) [inline scalajs/src/main/scala/org/scalablytyped/converter/internal/scalajs/flavours/GenCompanions.scala:58]: Can you flip these two `if`s as well? `if (enableLongApplyMethod) props else ...`

**Review by oyvindberg — APPROVED** [review]
Thanks!

I'll merge and publish a new release within a day or two. If you have time for the comment I left before I'd appreciate it :)

File: scalajs/src/main/scala/org/scalablytyped/converter/internal/scalajs/flavours/GenCompanions.scala (hunk 1 above)
Comment (Asamsig) [reply scalajs/src/main/scala/org/scalablytyped/converter/internal/scalajs/flavours/GenCompanions.scala:58]: Fixed.

**Comment by Asamsig** [comment]
I had mistakenly only run scalafmt for one module, I just ran it for the whole project and amended.

**Pull request merged**